### DISCUSSION TITLE
Version v1.2.1 Released!

### DISCUSSION THREAD

**Discussion by hollowaykeanho** [ZORALab/Hestia#401] [Announcements]
Minor patch against version 1.2.0. Its main focuses are to perform critical bug fixes discovered from the deployment of its predecessor and increase Hugo performances. More info at its documentation website.

| Language      | Location |
|:-------------------|:--------------|
| English           | https://hestia.zoralab.com/en/releases/v1p2p1/ |
| 华语（国际） | https://hestia.zoralab.com/zh-hans/releases/v1p2p1/ |

<hr /><em>This discussion was created from the release <a href='https://github.com/ZORALab/Hestia/releases/tag/v1.2.1'>Version v1.2.1</a>.</em>

**Comment by JeanShuralyov** [comment]
Yippi! That's way too fast. XD
